app/assets/javascripts/app/views/hovercard_view.js

```diff
@@ -94,14 +95,20 @@ app.views.Hovercard = app.views.Base.extend({
 
   _populateHovercard: function() {
     var href = this.href();
-    href += "/hovercard.json";
+    href += ".json";
```

Comment:
Are you sure you want to use the big person-json for the hovercards? Because this also contains the `bio` (and some other unused fields) and can be many times bigger than the hovercard.json.